src/popup-view-model.ts

```diff
@@ -85,9 +87,10 @@ export class PopupBaseViewModel extends Base {
     this.setupModel(model);
   }
 
-  constructor(model: PopupModel) {
+  constructor(model: PopupModel, environment: ISurveyEnvironment) {
```

Comment:
We always have access to survey settings. We can change it with
`this.environment = settings.environment`
So this pr will not break survey-creator and other dependents